FreeRTOS/FreeRTOS
# Update on target test for passive idle hook

Update for passive idle task hook

Description
-----------
Update on target test for passive idle hook

Test Steps
-----------
Before this PR there will be compilation error
Test steps:
```
cd ./FreeRTOS/FreeRTOS/Test/Target/boards/pico
cmake -S . -B build
cd build
make -j
```

Error message:
```
  179 |         #error Missing definition:  configUSE_PASSIVE_IDLE_HOOK must be defined in FreeRTOSConfig.h as either 1 or 0.  See the Configuration section of the FreeRTOS API documentation for details.

```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
